tests/subsys/net/lib/fota_download/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -5,4 +5,5 @@
 #
 
 # Disable this since we write our own mock for 'spm_s0_active()'
-CONFIG_SPM_SECURE_SERVICES=n
+CONFIG_SPM_SERVICE_S0_ACTIVE=n
+CONFIG_SPM_SERVICE_BUSY_WAIT=n
```

Comment:
Nit: do we need to disable this? The commit message states that we only disable the s0 active. Only fix this if you are pushing anyway :)

Reply:
It should be disabled for tests according to it config but we don't set CONFIG_TESTS=y in the sample.